RELEASE_NOTES.md

```diff
@@ -1,5 +1,33 @@
 # Release Notes
 
+## Changes in splinter 0.6.1
+
+### Highlights
+
+* Add a state migrate command for migrating state between database backends.
+
+* Update stores to support write-exclusivity for database backends
```

Comment:
Some details should be included under the various sections

Reply:
Fixed